readme.md

```diff
@@ -783,6 +754,26 @@ Default: `SIGTERM`
 
 Signal value to be used when the spawned process will be killed.
 
+#### forceKillAfterTimeout
```

Comment:
Since we're moving the option to the top-level, should this take this opportunity to rename it to `gracefulExit`? I am wondering whether `forceKillAfterTimeout` might lead to some confusion with the other `timeout` option which is quite different.